hgtv/views/video.py

```diff
@@ -270,27 +292,86 @@ def video_view(videopath):
+    else:
+        user['logged_in'] = False
+        user['login_url'] = url_for('login')
+
+    return dict(channel=channel, playlist=playlist,
+        video=video, speakers=speakers, user=user)
+
+
+def handle_edit_video(data):
+    playlist = data['playlist']
+    channel = data['channel']
+    video = data['video']
+    current_speakers = [speaker.userid for speaker in video.speakers]
+    form = VideoEditForm(obj=video)
+    if request.method == 'GET':
+        html_form = render_form(form=form, title="Edit Video", submit=u"Save",
+            cancel_url=video.url_for(), ajax=False, with_chrome=False)
+        return jsonify(video=get_video_details(channel, playlist, video), form=html_form)
+    if form.validate():
+        form.populate_obj(video)
+        if not playlist.name:
+            video.make_name()
+        if video.video_url != form.video_url.data:
+            try:
+                process_video(video, new=False)
+            except (DataProcessingError, ValueError) as e:
+                return make_response(jsonify(status='error', errors={'error': [e.message]}), 400)
+        if video.slides_url != form.slides_url.data:
+            try:
+                process_slides(video)
+                if video.video_slides_mapping:
+                    video.video_slides_mapping_json = make_presentz_json(video, json.loads(video.video_slides_mapping))
+            except (DataProcessingError, ValueError) as e:
+                return make_response(jsonify(status='error', errors={'error': [e.message]}), 400)
+        new_speakers = [new_speaker.userid for new_speaker in form.speakers.data]
+        for current_speaker in current_speakers:
+            # Remove speaker
+            if current_speaker not in new_speakers:
+                speaker_channel = Channel.query.filter_by(userid=current_speaker).first()
+                if speaker_channel:
+                    speaker_playlist = speaker_channel.playlist_for_speaking_in()
+                    if speaker_playlist:
+                        speaker_playlist.videos.remove(video)
+        for new_speaker in new_speakers:
+            # Add speaker
+            if new_speaker not in current_speakers:
+                userinfo = lastuser.getuser_by_userid(new_speaker)
+                if userinfo:
+                    speaker_channel = Channel.query.filter_by(userid=new_speaker).first()
+                    if speaker_channel is None:
+                        # Create a channel for this speaker. They have never logged in to hasgeek.tv
+                        # at this point, but when they do, the channel will be waiting for them
+                        speaker_channel = Channel(userid=userinfo['userid'],
+                                                  name=userinfo['name'] or userinfo['userid'],
+                                                  title=userinfo['title'],
+                                                  type=CHANNEL_TYPE.PERSON)
```

Comment:
Instead of putting them like this, put as many params as possible in single line and then go to second line - 
```
speaker_channel = Channel(userid=userinfo['userid'], name=userinfo['name'] or userinfo['userid'],
    title=userinfo['title'], type=CHANNEL_TYPE.PERSON)
```